.github/workflows/project-automations.yml

```diff
@@ -6,7 +6,7 @@ on:
       - reopened
       - closed
       - labeled
-  pull_request_target:
+  pull_request:
```

Comment:
In following the second example from [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request_target-workflow-based-on-the-head-or-base-branch-of-a-pull-request) the pull_request_target would be the base branch and we need to look at the head branch

Reply:
ooo ok nice!

Reply:
@NicholasBoll This would be accurate here since I am looking at the PRs that are not from a fork.
I want to specifically look at the head branch of PRs like [merge/master-into-prerelease/minor](https://github.com/Workday/canvas-kit/pull/2024), [merge/support-into-master](https://github.com/Workday/canvas-kit/pull/2023), and [dependabot/**](https://github.com/Workday/canvas-kit/pull/1972).

Reply:
I'd still leave it a `pull_request_target`. As far as I know, project actions should always work on metadata and never the workspace, so why limit this workflow to only work on non-forked pull requests?

Reply:
I guess this action specifically is only for the automated branches. In the future we might add actions for a forked pull request, and then would have the `pull_request_target` trigger event